src/aggregator/aggregator/aggregator.go

```diff
@@ -1066,23 +1066,27 @@ type aggregatorTickMetrics struct {
 	duration         tally.Timer
 	standard         tickMetricsForMetricCategory
 	forwarded        tickMetricsForMetricCategory
+	timed            tickMetricsForMetricCategory
 }
 
 func newAggregatorTickMetrics(scope tally.Scope) aggregatorTickMetrics {
 	standardScope := scope.Tagged(map[string]string{"metric-type": "standard"})
 	forwardedScope := scope.Tagged(map[string]string{"metric-type": "forwarded"})
+	timedScope := scope.Tagged(map[string]string{"metric-type": "timed"})
```

Comment:
unrelated to flushing, but noticed this type was missing when I was looking at the existing active metrics.